lms/templates/sudo/sudo.html

```diff
@@ -1,21 +1,25 @@
 {% extends "main_django.html" %}
-{% load i18n %}
+{% load i18n third_party_auth_links %}
 
 {% block body %}
-<section style="margin: 0 auto; width: 480px; padding: 50px;">
+<section class="sudo" style="margin: 0 auto; width: 480px; padding: 50px;">
     <div class="inner-wrapper">
         <header>
           <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
         </header>
         <hr />
-        <div style="margin: 0px auto; width: 218px;">
+        <div style="margin: 0px auto; width: 320px;">
             <form class="sudo-form" method="post">{% csrf_token %}
             {{ form.as_p }}
             <p>
```

Comment:
Please add "field" class to this p tag to use default input and label styles in _account.scss